.changes/linux-monitor-size.md

```diff
@@ -0,0 +1,5 @@
+---
+"tao": minor
```

Comment:
Should be `patch`

Reply:
and that's why i should actually look at the stuff i copy paste, sorry...